### ISSUE TITLE
how to deal with machine-generated labels of openimages ?

### ISSUE THREAD

**Issue by tangsipeng** [xingyizhou/UniDet#6]
how to deal with machine-generated labels of openimages ?

**Comment by LigZhong** [comment]
The images I downloaded does not match the label files either. Could anyone give a hit which version of data I should download.

**Comment by tangsipeng** [comment]
and i found many  missing labels，such like 88fb29e99f00706c，i draw all labels that human‘s related. 
![图片](https://user-images.githubusercontent.com/4188431/121132138-564aeb00-c863-11eb-914d-46bc354c0b8d.png)

**Comment by tangsipeng** [comment]
> 
> 
> The images I downloaded does not match the label files either. Could anyone give a hit which version of data I should download.

as the paper says ,2019 release of OpenImages

**Comment by LigZhong** [comment]
@tangsipeng Yes, I downloaded the version as indicated but found a image size mismatch error. And the classification accuracy curve is also wired.

**Comment by xingyizhou** [comment]
Hi,
  Sorry for the delayed response. I am not aware of the machine-generated label issue in OpenImages. It is likely a version mismatch --- as mentioned above, we used the Challenge-2019 release. I am aware of the size mismatch issue, but don't have a good solution (even though I have tried re-computing the image size when generating a new json file). I just simply comment out the assert when checking the size, it doesn't affect the training at all.
